silq/analysis/fit_toolbox.py

```diff
@@ -298,7 +306,10 @@ def add_to_plot(self,
             **{self.sweep_parameter: x_vals_full})
         x_vals_full *= xscale
         y_vals_full *= yscale
-        plot_kwargs = {**self.plot_kwargs, **kwargs}
+
+        kwargs = cbook.normalize_kwargs(kwargs, mlines.Line2D)
+        plot_kwargs = self.default_plot_kwargs
+        plot_kwargs.update(**kwargs)
```

Comment:
Don't these two lines change `Fit.default_plot_kwargs`? I would use `plot_kwargs = {**self.default_plot_kwargs, **kwargs}`, which should prioritize kwargs from the second dict

Reply:
and perhaps linewdith / lw should be treated separately... If a user also passes the kwarg `lw`, it would cause issues

Reply:
Good catch, this is still a WIP, I was looking for a better solution than this. But I think I can combine things while maintaining clarity.